tests/share/normalize/factories.py

```diff
@@ -339,6 +343,12 @@ class ThroughTagsFactory(ShareObjectFactory):
     # creative_work = factory.SubFactory(AbstractCreativeWorkFactory)
 
 
+class ThroughSubjectsFactory(ShareObjectFactory):
```

Comment:
I copied `ThroughTagsFactory` three lines up, but it's part of the automagic `GraphContructor` [sic] which [looks for factories by name](https://github.com/aaxelb/SHARE/blob/3b059751dd229e9b0f1e218cf6022f4d4f093494/tests/share/normalize/factories.py#L126). The subfactories are commented out because these through-factories are only used indirectly, when both the work and tag/subject are given.